### PR TITLE
Bugfix: should return error immediately in ecma_op_create_typedarray

### PR DESCRIPTION
**Pull request by jiangzidong**
Related issue: #1936

JerryScript-DCO-1.0-Signed-off-by: Zidong Jiang zidong.jiang@intel.com

### CODE REVIEW

File: tests/jerry/es2015/regression-test-issue-1936.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+// distributed under the License is distributed on an "AS IS" BASIS
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+var name = "";
+
+try
+{
+	Object.defineProperties(constructor.isExtensible, {a: Object.getOwnPropertyDescriptor(Uint8ClampedArray, "length")})
```
Comment (LaszloLango) [inline tests/jerry/es2015/regression-test-issue-1936.js]: Please remove tab characters

**Comment by jiangzidong** [comment]
fixed @LaszloLango

**Comment by akosthekiss** [comment]
What the heck just happened? Master is failing on Travis after this PR got merged. Everything was green on the PR before.

**Comment by jiangzidong** [comment]
@akosthekiss It's my fault. In the test case, I use Date.now() as the input arg for typedarray. But the result "Date.now()" will change by time. So last time, the case can pass the test, but this time, it can't.

**Comment by jiangzidong** [comment]
Do you want me to submit a patch? or can you revert the commit? @akosthekiss

**Pull request merged**